datadog_checks_base/tox.ini

```diff
@@ -2,7 +2,7 @@
 minversion = 2.0
 skip_missing_interpreters = true
 envlist =
-    py{27,36,37}
+    py{27,37}
```

Comment:
Our ci won't support 3.7 until https://github.com/DataDog/integrations-core/pull/2973 is fixed